From b85d6fdeb0d30c3fdf5a38f89c794da8e3776886 Mon Sep 17 00:00:00 2001
From: Alberto Milone <alberto.milone@canonical.com>
Date: Mon, 26 Oct 2015 12:17:08 +0100
Subject: [PATCH 1/1] Force gcc to 4.9 instead of 5.x

This is a workaround, and it should be dropped as soon as AMD
fixes the actual problem in the driver.
---
 make.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make.sh b/make.sh
index 1607d2e..7e20b20 100755
--- a/make.sh
+++ b/make.sh
@@ -56,7 +56,7 @@ set_GCC_version () {
 }
 
 if [ -z "${CC}" ]; then 
-	CC=gcc
+		CC=gcc-4.9
 	#set_GCC_version
 fi
 
-- 
1.9.1

